src/misc/support.cpp

```diff
@@ -399,32 +401,67 @@ const std_fs::path &GetExecutablePath()
 	return exe_path;
 }
 
-std_fs::path GetResource(const std_fs::path &name)
+std_fs::path GetResourcePath(const std_fs::path &name)
 {
 #if defined(MACOSX)
 	const auto bundled_path = GetExecutablePath() / "../Resources";
+	const auto outer_bundled_path = GetExecutablePath() / "../../Resources";
 #else
 	const auto bundled_path = GetExecutablePath() / "resources";
+	const auto outer_bundled_path = GetExecutablePath() / "../resources";
 #endif
-	const auto parents = {bundled_path,
+	const auto parents = {bundled_path, outer_bundled_path,
 	                      // macOS, POSIX, and even MinGW/MSYS2/Cygwin:
 	                      std_fs::path("/usr/local/share/dosbox-staging"),
 	                      std_fs::path("/usr/share/dosbox-staging"),
 	                      std_fs::path(CROSS_GetPlatformConfigDir())};
 
 	// return the first existing resource
+	std::error_code ec;
 	for (const auto &parent : parents) {
 		const auto resource = parent / name;
-		if (std_fs::exists(resource)) {
+		if (std_fs::exists(resource, ec)) {
 			return resource;
 		}
 	}
 	return std_fs::path();
 }
 
-std_fs::path GetResource(const std_fs::path &subdir, const std_fs::path &name)
+std_fs::path GetResourcePath(const std_fs::path &subdir, const std_fs::path &name)
 {
-	return GetResource(subdir / name);
+	return GetResourcePath(subdir / name);
+}
+
+std::vector<uint8_t> LoadResource(const std_fs::path &name,
+                                  ResourceImportance importance)
```

Comment:
argh, I'm not even following my own feedback here!
I forgot to mark `ResourceImportance` as `const` in both the `LoadResource` interfaces. 
(apologies for tacking this on; thanks @FeralChild64 !)

Reply:
Done.